public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -115,7 +121,7 @@ export function runPartitionedQuery(datasource: LokiDatasource, request: DataQue
       return;
     }
 
-    datasource
+    smallQuerySubsciption = datasource
```

Comment:
I think the name is a little bit confusing.
```suggestion
    subQuerySubsciption = datasource
```